src/AspNet.Security.OAuth.SuperOffice/SuperOfficeAuthenticationHandler.cs

```diff
@@ -38,13 +38,20 @@ public partial class SuperOfficeAuthenticationHandler : OAuthHandler<SuperOffice
     {
         var contextId = await ProcessIdTokenAndGetContactIdentifierAsync(tokens, properties, identity);
```

Comment:
It would be nicer to just deconstruct this:

```csharp
(string tenantId, string webApiUrl) = await ...
```

Reply:
I like this suggestion! Thanks.